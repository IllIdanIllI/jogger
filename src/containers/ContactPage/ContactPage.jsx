import React from 'react';
import './ContactPage';

const ContactPage = () => {
    return <div>Contact</div>;
};

export default ContactPage;
