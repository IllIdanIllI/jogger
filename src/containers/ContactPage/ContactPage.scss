.contact-container {
    top: 116px;
    text-align: center;
}