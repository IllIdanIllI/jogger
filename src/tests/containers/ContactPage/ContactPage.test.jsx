import React from 'react';
import { mount } from 'enzyme';
import configureStore from 'redux-mock-store';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import ContactPage from '../../../containers/ContactPage';

describe('<ContactPage /> unit test', () => {
    const initialState = {
        authentication: { isAuthenticated: true },
        jogStore: { isFilterActive: false },
    };
    const mockStore = configureStore();
    let store, wrapper;

    beforeEach(() => {
        store = mockStore(initialState);
        wrapper = mount(
            <Provider store={store}>
                <Router>
                    <ContactPage />
                </Router>
            </Provider>
        );
    });
    it('should be valid text', () => {
        expect(wrapper.find('.contact-container').text()).toEqual(
            'For mo information call: 102'
        );
    });
    it('Match snap', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
