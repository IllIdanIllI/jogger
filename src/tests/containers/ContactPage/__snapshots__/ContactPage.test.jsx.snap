// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContactPage /> unit test Match snap 1`] = `ReactWrapper {}`;
